### ISSUE TITLE
moveItem crashes addon

### ISSUE THREAD

**Issue by hazcod** [davecra/easyEWS#16]
Version `.20`.

```js
function deleteEmail(boxItem)
{
  easyEws.moveItem(boxItem.itemId, deletedFolderName, function() {
    console.log('moved');
  }, function(err){ console.log(err); });;
}
```

Returns following error before crashing the add-on:
![image](https://user-images.githubusercontent.com/5222512/123383950-163e7480-d594-11eb-96ea-e13e30eda20a.png)

**Comment by hazcod** [comment]
Created https://github.com/davecra/easyEWS/pull/17

**Comment by davecra** [comment]
Fixed this. Verified it is resolved in 1.21, to be released soon.

**Comment by davecra** [comment]
Pushed and verified in NPM and on CDN.
